modules/data-entry/src/main/frontend/src/questionnaire/FormEntry.jsx

```diff
@@ -120,11 +122,11 @@ let displaySection = (sectionDefinition, path, depth, existingAnswer, key) => {
  * @returns a React component that renders the section
  */
  export default function FormEntry(props) {
-  let { classes, entryDefinition, path, depth, existingAnswers, keyProp } = props;
+  let { classes, entryDefinition, path, depth, existingAnswers, keyProp, answersTracker, didGrow } = { ...{answersTracker: null, didGrow: null}, ...props };
```

Comment:
```suggestion
  let { classes, entryDefinition, path, depth, existingAnswers, keyProp, answersTracker, didGrow } = props;
```
If we change the check in the comment below, it's possible to work around using null when you mean undefined